_posts/2024-04-18-SceneMaps Bug Fix.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+# If your post title is longer or more complicated
+# than can be represented in the filename, uncomment the following line
+# and specify a custom title
+# title:  "SceneMaps Bug Fix"
+
+# Uncomment only one of the below categories
+categories: 
+ Bug Fix
```

Comment:
Seems like the hyphen here may have gotten deleted at some point and jekyll thinks this is two categories, not one. Super minor

```suggestion
- Bug Fix
```